client/feature/ProfilePhotoAndName.js

```diff
@@ -36,6 +38,46 @@ export const ProfileOldStuff = {
     }
 }
 
+export function VerificationBadge({persona, size}) {
+    const datastore = useDatastore();
+    const fbUser = datastore.getFirebaseUser();
+    const verificationStatus = persona?.verificationStatus ?? null;
+    
+    if (fbUser?.displayName !== persona?.name) {
```

Comment:
This will always be true, except for the badge of the currently logged in user. I don't think that's what you want.

I think you probably want to check if the name starts with a capital letter - and maybe also enforce that for real names in the setup screen.